integration/pom.xml

```diff
@@ -21,32 +21,27 @@
   <packaging>pom</packaging>
   <name>Alluxio Integration</name>
   <description>Parent POM for different integrations with Alluxio</description>
+  <modules>
+    <!-- exclude checker from hadoop-1 profile as yarn is not available in Hadoop 1.x -->
+    <module>fuse</module>
+  </modules>
 
   <properties>
     <!-- The following paths need to be defined here as well as in the parent pom so that mvn can -->
     <!-- run properly from sub-project directories -->
     <build.path>${project.parent.basedir}/build</build.path>
   </properties>
   <profiles>
-    <profile>
-      <id>hadoop-1</id>
-      <modules>
-        <!-- exclude checker from hadoop-1 profile as yarn is not available in Hadoop 1.x -->
-        <module>fuse</module>
-      </modules>
-    </profile>
     <profile>
       <id>hadoop-2</id>
       <modules>
         <module>checker</module>
```

Comment:
unrelated to this:  maybe we can remove checker module --- not really see this used

Reply:
agree, user just need to build it separately.